deeptime/markov/msm/tram/_tram.py

```diff
@@ -0,0 +1,708 @@
+import copy
+import warnings
+from typing import Optional
+
+from deeptime.markov.msm import MarkovStateModelCollection
+from deeptime.markov import TransitionCountEstimator, TransitionCountModel
+from deeptime.markov._base import _MSMBaseEstimator
+from deeptime.util import types
+# from _tram_bindings import tram
+from deeptime.markov._tram_bindings import tram
```

Comment:
i think tram bindings should live here and be imported by `from . import _tram_bindings`

Reply:
Do you mean change the folder structure and move the tram bindings into the folder where the .py file lives? Or just reorganize the imports?

Reply:
i mean to move the tram bindings into the tram module (ie move `deeptime.markov.bindings.tram` to `deeptime.markov.msm.tram.bindings`)